src/components/NewGalaxyModal.js

```diff
@@ -83,23 +99,29 @@ export const NewGalaxyModal = _.flow(
       }],
       ['paused', () => {
         return h(Fragment, [
-          h(ButtonPrimary, { style: { marginRight: 'auto' }, onClick: () => setViewMode('deleteWarn') }, ['Delete']),
-          h(ButtonSecondary, { style: { marginRight: '1rem' }, onClick: resumeGalaxy }, ['Resume'])
+          deleteButton(),
+          resumeButton()
         ])
       }],
       [Utils.DEFAULT, () => !app ?
-        h(ButtonPrimary, { onClick: () => setViewMode('createWarn') }, ['Next']) :
+        h(ButtonPrimary, { disabled: false, onClick: () => setViewMode('createWarn') }, ['Next']) :
         Utils.switchCase(app.status,
           ['RUNNING', () => h(Fragment, [
-            deleteButton,
-            h(ButtonSecondary, { style: { marginRight: '1rem' }, onClick: () => { pauseGalaxy() } }, ['Pause']),
-            h(ButtonPrimary, { onClick: () => setViewMode('launchWarn') }, ['Launch Galaxy'])
+            deleteButton(),
+            pauseButton(),
+            h(ButtonPrimary, { disabled: false, onClick: () => setViewMode('launchWarn') }, ['Launch Galaxy'])
           ])],
           ['STOPPED', () => h(Fragment, [
-            deleteButton,
-            h(ButtonPrimary, { style: { marginRight: '1rem' }, onClick: () => { resumeGalaxy() } }, ['Resume'])
+            h(ButtonSecondary, { disabled: true, style: { marginRight: 'auto' }, tooltip: 'Cloud Compute must be resumed first.', onClick: () => setViewMode('deleteWarn') }, ['Delete']),
+            resumeButton()
           ])],
-          ['ERROR', () => deleteButton]
+          ['ERROR', () => deleteButton],
+          [Utils.DEFAULT, () => {
+            return h(Fragment, [
+              deleteButton(true),
```

Comment:
I think this should be `false`. Otherwise it is not allowing the user to delete the galaxy if it is errorred.

Reply:
Seems the `ERROR` case is handled above on L118?

Reply:
Totally! Thanks for catching this 🙏🏼

Reply:
Oh actually, the `ERROR` case is above and does have a `deleteButton`. The one you pointed out is for undelete-able cases.

Reply:
(For some reason Rob's comment above hadn't shown up on my screen when I wrote the above replies  🤷🏼‍♂️ )